content/en/account_management/rbac/role_api.md

```diff
@@ -17,162 +18,375 @@ The Roles API can be used to create and manage Datadog roles, what global permis
+##### Expected Response code: 
+
+```sh
+HTTP/2 200
+```
+
+##### Example:
 
-Example:
 
 ```sh
-curl -X GET "https://app.datadoghq.com/api/v1/roles?api_key=${API_KEY}&application_key=${APP_KEY}"
+curl -X GET "https://app.datadoghq.com/api/v2/roles" -H "DD-API-KEY: ${API_KEY}" -H "DD-APPLICATION-KEY: ${APP_KEY}"
+
 
 # Response:
-# [{
-#   "id": <number>,
-#   "name": <string>,
-#   "uuid": <string>
-#  }, ...]
+# {
+#     "meta": {
+#         "page": {
+#             "total_count": 7
+#         }
+#     },
+#     "data": [
+# 	    {
+#             "type": "roles",
+#             "id": $ROLE_UUID,
+#                 "attributes": {
+#                 "created_at": "2000-02-29T16:50:43.607749+00:00",
+#                 "user_count": 2122,
+#                 "modified_at": "2000-02-29T16:50:43.607749+00:00",
+#                 "uuid": $ROLE_UUID,
+#                 "name": $ROLE_NAME
+#             },
+#             "relationships": {
+#                 "permissions": {
+#                     "data": [
+#                         {
+#                             "type": "permissions",
+#                             "id": $PERMISSION_UUID
+#                         },
+#                         {
+#                             "type": "permissions",
+#                             "id": $PERMISSION_UUID
+#                         }
+#                     ]
+#                 }
+#             }
+#         }
+#     ]
+# }
 ```
 
 ### Get One Role
 
 Description: Returns a specific role, including its name and uuid.  
 Method: GET  
-Endpoint: `api/v1/roles/$ROLE_UUID`  
+Endpoint: `api/v2/roles/$ROLE_UUID`  
 Required Payload: No Payload  
-Example:
 
+##### Response code: 
 ```sh
-curl -X GET "https://app.datadoghq.com/api/v1/roles/${ROLEUUID}?api_key=${API_KEY}&application_key=${APP_KEY}"
+HTTP/2 200
+```
+
+##### Example:
+
+```sh
+curl -X GET "https://app.datadoghq.com/api/v2/roles/$ROLE_UUID" -H "DD-API-KEY: ${API_KEY}" -H "DD-APPLICATION-KEY: ${APP_KEY}"
 
 # Response:
 # {
-#  "id": <number>,
-#  "name": <string>,
-#  "uuid": <string>
+# 	"data": {
+#             "type": "roles",
+#             "id": $ROLE_UUID,
+#                 "attributes": {
+#                 "created_at": "2000-02-29T16:50:43.607749+00:00",
+#                 "user_count": 2122,
+#                 "modified_at": "2000-02-29T16:50:43.607749+00:00",
+#                 "uuid": $ROLE_UUID,
+#                 "name": $ROLE_NAME
+#             },
+#             "relationships": {
+#                 "permissions": {
+#                     "data": [
+#                         {
+#                             "type": "permissions",
+#                             "id": $PERMISSION_UUID
+#                         },
+#                         {
+#                             "type": "permissions",
+#                             "id": $PERMISSION_UUID
+#                         }
+#                     ]
+#                 }
+#             }
+#         }
 # }
 ```
 
 ### Create Role
```

Comment:
does this return the newly created role as the response? if so we should include this

Reply:
Done